src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,17 @@
-export const TodoList = () => {};
+import { ToDo } from '../../types/types';
+import { TodoInfo } from '../TodoInfo';
+import React from 'react';
+
+type Props = {
+  todos: ToDo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="TodoList">
+      {todos.map((todo, indx) => {
+        return <TodoInfo key={indx} todo={todo} />;
```

Comment:
Using the index `indx` as a key can lead to issues with component re-rendering. Consider using a unique identifier from the `todo` object, such as `todo.id`, as the key instead.